plugins/webapi/plugin.go

```diff
@@ -75,6 +78,19 @@ func configure(plugin *node.Plugin) {
 		}
 	}
 
+	// Load whitelisted addresses
+	whitelist := append([]string{"127.0.0.1", "::1"}, parameter.NodeConfig.GetStringSlice("api.whitelistedAddresses")...)
+	if len(whitelist) > 0 {
+		for _, entry := range whitelist {
+			_, ipnet, err := cnet.ParseCIDROrIP(entry)
+			if err != nil {
+				log.Warnf("Invalid whitelist address: %s", entry)
+			} else {
+				whitelistedNetworks = append(whitelistedNetworks, ipnet.IPNet)
+			}
```

Comment:
We can remove the `else` here
```suggestion
			if err != nil {
				log.Warnf("Invalid whitelist address: %s", entry)
				continue
			} 
		        whitelistedNetworks = append(whitelistedNetworks, ipnet.IPNet)
```